src/directives/formly-form.js

```diff
@@ -16,7 +16,7 @@ module.exports = ngModule => {
       restrict: 'E',
       template: function(el, attrs) {
         /* jshint -W033 */ // this because jshint is broken I guess...
-        var rootEl = attrs.rootEl || 'ng-form';
+        var rootEl = attrs.rootEl || 'form';
```

Comment:
@kentcdodds Would you be fine with this change:
``` javascript
        var formTag = (attrs.rootEl || 'ng-form').split(' ');
        var formEl = formTag[0];
        var formAtt = formTag.slice(1).join(' ');

        return `
          <${formEl} ${formAtt} class="formly">
          </${formEl}>
        `;
```
That way, the default is still 'ng-form', and users can specify things like `root-el="form"` or `root-el="div ng-form"`.